molecule/default/playbook.yml

```diff
@@ -14,6 +14,11 @@
     admin_password: sup3rs3cur3pa55w0rdf0rt3st1ng
     # change the default prefix to easily identify test containers / services
     hub_service_prefix: molecule_test_hub
+    #jwt
+    jwt_privatekey: jwt-key
+    jwt_publickey: jwt-key.pub
+    jwt_credentials_path: "{{ playbook_dir }}/sample_cert/jwt"
+    jwt_remote_certificate_path: "{{ playbook_dir }}/sample_cert/jwt" # Because docker in docker uses the host system instead of the actual remote
```

Comment:
comments always above the line to comment on (we don't do EOL comments as they quickly become invisible once the code line itself gets too long